lua/nerdy/config.lua

```diff
@@ -3,14 +3,29 @@ local M = {}
 ---@class nerdy.config
 ---@field max_recents integer : Max number of recent icons to keep
 ---@field add_default_keybindings boolean : Whether to add default keybindings
----@field copy_to_clipboard boolean : -- Copy glyph to clipboard instead of inserting
+---@field output_location string|nil : Register to output icon to (nil for insert in place)
+---@field copy_to_clipboard boolean|nil : @deprecated Use output_location instead
 M.config = {
     max_recents = 30,
     add_default_keybindings = true,
-    copy_to_clipboard = false,
+    output_location = nil,
```

Comment:
we will not be getting rid of this config, it will work in conjunction with the new one, copying this from the issue:

if copy_to_clipboard is true then it will copy to the system clipboard

If copy_to_clipboard is true **and** an output_register string is provided in the config

then instead of copying to the clipboard register, it copies to the configured output_register

this way, we get the new enhancement without breaking backwards compat